tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/UsernamePasswordIntegrationTests.cs

```diff
@@ -83,6 +84,14 @@ public async Task ROPC_ADFSv4Federated_Async()
             await RunHappyPathTestAsync(labResponse).ConfigureAwait(false);
         }
 
+        [TestMethod]
+        public async Task ROPC_ADFSv4Federated_WithMetadata_Async()
+        {
+            var labResponse = await LabUserHelper.GetAdfsUserAsync(FederationProvider.AdfsV4, true).ConfigureAwait(false);
+            string federationMetadata = File.ReadAllText(@"federationMetadata.xml").ToString();
```

Comment:
Can you run the test from both .net classic and .net core? I remember that for unit tests the paths were different...

Reply:
Yes. The xml file is copied to the output build directory so they will be available for both platforms